packages/lib/src/utils/errors/DeployError.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+export class DeployError extends Error {
```

Comment:
How about creating a single DeployError that accepts an object, and assigns it to `this`? So we can have a generic DeployError that we use for every circumstance.